Dockerfile

```diff
@@ -0,0 +1,23 @@
+# BUILD_TYPE can have these values: local, test or prod

```

Comment:
Est-ce que ces commentaires sont encore pertinents?

Reply:
Yes, because those comments are for people who are not familiar with docker multi-stages builds that will have to interact with these shenanigans.